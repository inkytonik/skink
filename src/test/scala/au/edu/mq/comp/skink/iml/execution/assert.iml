def main(a): (r) {
  a = 0;
  assert(true);
  a = a + 1;
  print(a);
  assert(false);
  a = a + 10;
  print(a);
  r = 0;
}
