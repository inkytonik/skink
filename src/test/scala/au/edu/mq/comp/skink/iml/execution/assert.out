1
assertion failed at line 6 column 10
exit code: 255
